addons/product/data/product_demo.xml

```diff
@@ -600,6 +626,21 @@
             <field name="image_1920" type="base64" file="product/static/img/product_product_d01-image.jpg"/>
         </record>
 
+        <record id="product_template_no_variant_attribute" model="product.template">
+            <field name="name">Outdoor dinning table</field>
+            <field name="categ_id" ref="product_category_outdoor_furniture"/>
+            <field name="list_price">589</field>
+            <field name="detailed_type">consu</field>
+            <field name="image_1920" type="base64" file="product/static/img/product_template_no_va-image.png"/>
+        </record>
+
+        <!-- the product template attribute lines have to be defined before creating the variants -->
+        <record id="product_template_no_variant_attribute_attribute_4_product_template_attribute_line" model="product.template.attribute.line">
+            <field name="product_tmpl_id" ref="product_template_no_variant_attribute"/>
+            <field name="attribute_id" ref="product_attribute_4"/>
+            <field name="value_ids" eval="[(6,0,[ref('product.product_attribute_value_7'), ref('product.product_attribute_value_8'), ref('product.product_attribute_value_9')])]"/>
```

Comment:
No need to specify the module if it is defined in the same one in a `ref` + spaces after `,`
```suggestion
            <field name="value_ids" eval="[(6, 0, [ref('product_attribute_value_7'), ref('product_attribute_value_8'), ref('product_attribute_value_9')])]"/>
```